components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -260,12 +258,12 @@ - (void)layoutSubviews {
       titleCenter.y = roundf((float)titleCenter.y);
 
     } else {
-      // Title only, center vertically.
-      // +1 for slight adjustment to font baseline for centered title labels.
-      const CGFloat centeredTitleYOffset = 1.0f;
-      titleCenter.y = contentBounds.size.height - kBottomPadding - titleSize.height +
-                      centeredTitleYOffset + ((float)titleSize.height / 2.0f);
-      titleCenter.y = roundf((float)titleCenter.y);
+      // UILabels should be aligned to pixel boundaries to avoid anti-aliasing blurriness
```

Comment:
We are calculating the center here.
For pixel-alignment do we need to ensure that the label.frame.offset is on a pixel boundary?
If that is the case, consider moving the rounding to after we set the frame below (label.bounds, label.center below).
Do we need to align the badge and image as well?

Reply:
I can move the rounding code to where the bounds/center are set. I wasn't planning to change the layout of the badge or image in this PR since I was only trying to bring the text-only style up to spec.  I didn't notice misaligned images when I tested, but I'll take another look.